## /bin/sh -e simulate dpatch for prepend_chlog.py
## conf_sample.dpatch by Jose Carlos Garcia Sogo <jsogo@debian.org>

Index: asterisk-1.2.27/Makefile
===================================================================
--- asterisk-1.2.27.orig/Makefile	2008-03-19 11:33:02.000000000 +0100
+++ asterisk-1.2.27/Makefile	2008-03-19 11:33:09.000000000 +0100
@@ -797,10 +797,6 @@
 			exit 1; \
 		fi; \
 	done
-	mkdir -p $(DESTDIR)$(ASTVARLIBDIR)/mohmp3 ; \
-	for x in sounds/moh/*.mp3 sounds/moh/LICENSE; do \
-		$(INSTALL) -m 644 $$x $(DESTDIR)$(ASTVARLIBDIR)/mohmp3 ; \
-	done
 	rm -f $(DESTDIR)$(ASTVARLIBDIR)/mohmp3/sample-hold.mp3
 	mkdir -p $(DESTDIR)$(ASTSPOOLDIR)/voicemail/default/1234/INBOX
 	:> $(DESTDIR)$(ASTSPOOLDIR)/voicemail/default/1234/unavail.gsm
